From a39526ba5d8e54145250200ea58f5c97954d85bd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20D=C3=B6singer?= <stefan@codeweavers.com>
Date: Fri, 30 Jul 2010 06:31:40 +0200
Subject: [PATCH 1/7] libwine: msvc hack

---
 libs/wine/loader.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libs/wine/loader.c b/libs/wine/loader.c
index 649aa221d1..b162d20e24 100644
--- a/libs/wine/loader.c
+++ b/libs/wine/loader.c
@@ -50,7 +50,7 @@
 #undef GetCurrentThread
 #include <pthread.h>
 #include <mach-o/getsect.h>
-#else
+#elif !defined(_MSC_VER)
 extern char **environ;
 #endif
 
-- 
2.24.0.windows.2

