From 3d1499e2f10fad3306e13e50d1d28e7ced76b300 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20D=C3=B6singer?= <stefan@codeweavers.com>
Date: Wed, 18 Dec 2013 14:54:03 +0100
Subject: [PATCH 2/7] includes: Add hacks to d3dcommon.idl to make midl happy.

---
 include/d3dcommon.idl | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/d3dcommon.idl b/include/d3dcommon.idl
index 5e38658f89..e7b33baac3 100644
--- a/include/d3dcommon.idl
+++ b/include/d3dcommon.idl
@@ -55,8 +55,9 @@ typedef enum _D3D_INCLUDE_TYPE
 [
     object,
     local,
+    uuid(a14d602c-1d84-413e-68b8-4fc55c362cec)
 ]
-interface ID3DInclude
+interface ID3DInclude : IUnknown
 {
     HRESULT Open(D3D_INCLUDE_TYPE include_type,
                  const char *filename,
-- 
2.21.0.windows.1

