From 6b78974d4e546d6e48e0ba4d31e8b546da51d05a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20D=C3=B6singer?= <stefan@codeweavers.com>
Date: Sun, 2 Jun 2019 17:32:30 +0200
Subject: [PATCH 7/7] include: HJack for d3d10_1.idl.

---
 include/d3d10_1.idl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/d3d10_1.idl b/include/d3d10_1.idl
index a37ff9ea3b..9b6f3fb86d 100644
--- a/include/d3d10_1.idl
+++ b/include/d3d10_1.idl
@@ -152,6 +152,6 @@ const UINT D3D10_1_SDK_VERSION = 0x20;
 cpp_quote("HRESULT WINAPI D3D10CreateDevice1(IDXGIAdapter*,D3D10_DRIVER_TYPE,")
 cpp_quote("    HMODULE,UINT,D3D10_FEATURE_LEVEL1,UINT,ID3D10Device1**);")
 
-[local] HRESULT __stdcall D3D10CreateDeviceAndSwapChain1(IDXGIAdapter *adapter, enum D3D10_DRIVER_TYPE driver_type,
+[local] HRESULT __stdcall D3D10CreateDeviceAndSwapChain1(IDXGIAdapter *adapter, UINT driver_type,
         HMODULE swrast, UINT flags, D3D10_FEATURE_LEVEL1 feature_level, UINT sdk_version,
         DXGI_SWAP_CHAIN_DESC *swapchain_desc, IDXGISwapChain **swapchain, ID3D10Device1 **device);
-- 
2.21.0.windows.1

